parsers/medleydb_parser.py

```diff
@@ -37,118 +38,133 @@
+                                                     email='rachel.bittner@nyu.edu')
     annot.annotation_metadata.annotator = {}
 
 
 def fill_instid_annotation_metadata(annot):
     """Fills the annotation metadata."""
-    annot.annotation_metadata.corpus = "MedleyDB"
-    annot.annotation_metadata.version = "1.0"
+    annot.annotation_metadata.corpus = CORPUS
+    annot.annotation_metadata.version = VERSION
     annot.annotation_metadata.annotation_tools = ""
     annot.annotation_metadata.annotation_rules = ""
     annot.annotation_metadata.validation = "None"
     annot.annotation_metadata.data_source = "Automatic Annotation"
-    annot.annotation_metadata.curator = pyjams.Curator(name="Juan P. Bello",
-                                                       email='jpbello@nyu.edu')
+    annot.annotation_metadata.curator = jams.Curator(name="Juan P. Bello",
+                                                     email='jpbello@nyu.edu')
     annot.annotation_metadata.annotator = {}
 
 
-def fill_melody_annotation(annot_fpath, melody_annot, mel_type):
+def fill_genre_annotation(genre):
+
+    ann = jams.Annotation(namespace='tag_open')
+    ann.append(time=0.0, duration=0.0, value=genre)
+    fill_genre_annotation_metadata(ann)
+    return ann
+
+def fill_melody_annotation(annot_fpath, mel_type):
     """Fill a melody annotation with data from annot_fpath."""
-    times, values = U.read_lab(annot_fpath, 2, delimiter=",")
-    U.fill_timeseries_annotation_data(times, values, None, melody_annot)
-    fill_melody_annotation_metadata(melody_annot, mel_type)
 
+    ann = jams.Annotation(namespace='pitch_hz')
+    df = pd.read_csv(annot_fpath, header=None, names=['time', 'frequency'])
+
+    for _, record in df.iterrows():
+        ann.append(time=record['time'],
+                   duration=0.0,
+                   value=record['frequency'])
+    ann.time = 0.0
+    ann.duration = df['time'].max()
+
+    fill_melody_annotation_metadata(ann, mel_type)
+
+    return ann
 
-def fill_instid_annotation(annot_fpath, instid_annot):
+def fill_instid_annotation(annot_fpath):
     """Fill an instrument id annotation with data from annot_fpath."""
-    start_times, end_times, inst_labels = U.read_lab(annot_fpath, 3, 
-                                                     delimiter=",", header=True)
-    U.fill_range_annotation_data(start_times, end_times, inst_labels, 
-                                 instid_annot)
-    fill_instid_annotation_metadata(instid_annot)
 
+    ann = jams.Annotation(namespace='tag_medleydb_instruments')
+    df = pd.read_csv(annot_fpath)
+
+    for _, record in df.iterrows():
+        ann.append(time=record['start_time'],
+                   duration=record['end_time'] - record['start_time'],
+                   value=record['instrument_label'])
+    ann.time = 0.0
+    ann.duration = df['end_time'].max()
+
+    fill_instid_annotation_metadata(ann)
+
+    return ann
 
```

Comment:
Yes, those annotations are in the ACTIVATION_CONF file. They don't really line up with the source ids directly - they're really a stem level annotation, rather than a track level one.